backend/tests/unit/core/test_annotation_unit.py

```diff
@@ -10,6 +11,28 @@ def test_annotation_unit_gets_missing_dependencies(annotation_unit_lmna):
     assert actual == ['HGNC_ID']
 
 
+def test_annotation_unit_ready_for_annotation(annotation_unit_lmna):
+    """Verifies if the annotation unit is ready for annotation"""
+
+    missing = annotation_unit_lmna.get_missing_dependencies()
+    missing_dependency = missing[0]
+    mock_genomic_unit_collection = Mock()
+    mock_genomic_unit_collection.find_genomic_unit_annotation_value = Mock()
+    annotation_value = mock_genomic_unit_collection.find_genomic_unit_annotation_value(
+        annotation_unit_lmna.genomic_unit, missing_dependency
+    )
+
+    actual = annotation_unit_lmna.ready_for_annotation(annotation_value, missing_dependency)
+    assert actual is True
+
+
+def test_annotation_unit_should_continue_annotation(annotation_unit_lmna):
```

Comment:
A case where there are missing dependencies and one where there aren't will help inform how the function is to be used.  Really close!

Reply:
I have a couple of questions about this, and will follow up with you about this in slack :)